{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -0,0 +1,56 @@
+PYTHON_VERSION := 3.11
+WHEEL_VERSION := 0.43.0
+
+.DEFAULT_GOAL := venv/bin/activate
+
+venv/bin/activate: requirements-dev.txt
```

Comment:
why do we need this `requirements-dev.txt` here?

Reply:
makefile will check if requirments file changed, if yes then venv will be rebuild :)